package.json

```diff
@@ -42,8 +42,8 @@
     "@pika/plugin-standard-pkg": "^0.9.2",
     "jest": "^26.4.2",
     "jq-cli-wrapper": "^1.6.1",
-    "semantic-release": "^17.1.1",
-    "typescript": "^4.0.2"
+    "semantic-release": "^17.4.1",
```

Comment:
known issue with this version: https://github.com/semantic-release/semantic-release/issues/1784#issuecomment-790022983